src/lib/api/date.js

```diff
@@ -25,7 +25,7 @@ export const toISO = date => {
  *  @returns {String}
  */
 export const toISODate = date => {
-  return date.toISODate();
+  return date instanceof DateTime ? date.toISODate() : date;
```

Comment:
Question for everyone @jrcastro2 @topless @FlorianCassayre @KonstantinaStoikou @kprzerwa, after the discussion we had, does it make sense to keep this util module? Wondering if it is not just easier to use `luxon` directly around instead of "wrapping" it here...

Reply:
I'm OK with removing it, however it means a bit of time spent on refactoring (please remove if it is not a time eating monster)